docs/content/concepts/ops-jobs-graphs/jobs-graphs.mdx

```diff
@@ -398,6 +398,40 @@ def alias():
     add_one.alias("second_addition")(add_one(return_one()))
 ```
 
+### Providing Top Level Inputs to a Job
+
+You can wire inputs from the top-level of a job to the constituent ops. Consider the following op and graph:
+
+```python file=/concepts/ops_jobs_graphs/jobs.py startafter=start_top_level_input_graph endbefore=end_top_level_input_graph
+from dagster import graph, op
+
+
+@op
+def op_with_input(x):
+    return do_something(x)
+
+
+@graph
+def wires_input(x):
+    op_with_input(x)
+```
+
+I can turn the graph into a job by calling <PyObject object="GraphDefinition" method="to_job" />, and can provide a value to the input `x` using the `input_values` argument.
```

Comment:
```suggestion
Turn the graph into a job by calling <PyObject object="GraphDefinition" method="to_job" /> and providing a value to the input `x` using the `input_values` argument:
```

Second person is okay in the docs, but let's refrain from first. I changed this to use an understood subject (`you`) and make it second person.